src/lib.rs

```diff
@@ -52,7 +52,7 @@ mod hstring;
 mod object;
 mod param;
 mod ref_count;
-mod runtime;
+pub mod runtime;
```

Comment:
This makes all runtime functions public which we probably don't won't.

Reply:
Yes, I didn't like this. I need to find a way to tuck this into the winrt-rs implementation.